packages/venia-ui/lib/components/ProductFullDetail/productFullDetail.js

```diff
@@ -12,7 +12,7 @@ import Button from '../Button';
 import Carousel from '../ProductImageCarousel';
 import FormError from '../FormError';
 import { fullPageLoadingIndicator } from '../LoadingIndicator';
-import Quantity from '../ProductQuantity';
```

Comment:
Oh also, can we throw a `@deprecated` label on the old `ProductQuantity` component? After this PR it will only be used in the legacy mini cart.

Reply:
There were two things I wanted to do as a followup, prefer to just merge this:

https://github.com/magento/pwa-studio/pull/2690#pullrequestreview-497817541

Reply:
Created [PWA-960](https://jira.corp.magento.com/browse/PWA-960), thanks for the reminder; would have missed actually creating that in the backlog.

Reply:
@tjwiebell please mention to me when you create that issue. I hope I can resolve that :D